src/v2/Components/LotTimer.tsx

```diff
@@ -0,0 +1,120 @@
+import { LotTimer_saleArtwork } from "v2/__generated__/LotTimer_saleArtwork.graphql"
+import { LotTimerQuery } from "v2/__generated__/LotTimerQuery.graphql"
+import { SystemContext } from "v2/System"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { useContext } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import * as React from "react"
+import { Flex, Text } from "@artsy/palette"
+import { useTimer } from "v2/Utils/Hooks/useTimer"
+import { get } from "v2/Utils/get"
+
+export interface LotTimerProps {
+  saleArtwork: LotTimer_saleArtwork
+}
+
+export const LotTimer: React.FC<LotTimerProps> = ({ saleArtwork }) => {
+  const { endAt } = saleArtwork
+
+  const startAt = get(saleArtwork, sa => sa.sale?.startAt)
+
+  const { hasEnded, time, hasStarted } = useTimer(endAt || "", startAt || "")
+
+  if (!endAt) {
+    return null
+  }
+
+  const timerCopy = getTimerCopy(time, hasStarted)
+
+  return (
+    <Flex alignItems="center" flexDirection="column">
+      <Text variant="md" color={"blue100"}>
+        {!hasEnded && <Text color={timerCopy.color}>{timerCopy.copy}</Text>}
+      </Text>
+
+      <Text variant="md" color={"black60"}>
+        {saleArtwork.formattedStartDateTime}
+      </Text>
+    </Flex>
+  )
+}
+
+export const LotTimerFragmentContainer = createFragmentContainer(LotTimer, {
+  saleArtwork: graphql`
+    fragment LotTimer_saleArtwork on SaleArtwork {
+      endAt
+      formattedStartDateTime
+      sale {
+        startAt
+      }
+    }
+  `,
+})
+
+export const LotTimerQueryRenderer = ({
+  saleArtworkID,
+}: {
+  saleArtworkID: string
+}) => {
+  const { relayEnvironment } = useContext(SystemContext)
+  return (
+    <SystemQueryRenderer<LotTimerQuery>
+      environment={relayEnvironment}
+      variables={{ saleArtworkID }}
+      query={graphql`
+        query LotTimerQuery($saleArtworkID: String!) {
+          saleArtwork(id: $saleArtworkID) {
+            ...LotTimer_saleArtwork
+          }
+        }
+      `}
+      render={({ props }) => {
```

Comment:
```
render={{ error, props }} => {
  if (error) {
    console.error(error)
  }
  if (!props) {
    return PLACEHOLDER
  }
  if (props.saleArtwork) {
    return ... 
  }
}
```

(Check out some of the query renderers in Apps/Home to get an idea of how its most recently been done)